docs/make.jl

```diff
@@ -5,6 +5,13 @@ ENV["CLIMATEMACHINE_SETTINGS_DISABLE_GPU"] = true
 # avoid problems with Documenter/Literate when using `global_logger()`
 ENV["CLIMATEMACHINE_SETTINGS_DISABLE_CUSTOM_LOGGER"] = true
 
+run_single_tutorial = false
+run_single_tutorial = run_single_tutorial && isempty(get(ENV, "CI", "")) # never use remotely
+run_single_tutorial && (tutorial_string_match = "tendency_specification_layer")
```

Comment:
Is this a particular tutorial? If so, it might be too hacky for me.

Reply:
Yes, it is, this is what users can change to grep the tutorial they're testing

Reply:
Should we set `tutorial_string_match = ""` by default or something?

Reply:
Maybe `"<tutorial_name>"`? Your choice. :)